charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -30,4 +30,77 @@ spec:
   ingress:
     - {}
   {{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-prometheus
+  namespace: kyverno
```

Comment:
This should use `{{ include "kyverno.namespace" }}` rather than hardcoding the namespace.